### ISSUE TITLE
Wacatac.B!ml Trojan Virus Detected

### ISSUE THREAD

**Issue by Epik-B0mber** [kettingpowered/kettinglauncher#23]
So I've been trying to set up a server with the 1.3.3 launcher, and uhhh... well, it keeps saying that a trojan virus is detected, and that just deletes it. This doesn't happen with the ketting-1.20.2-48.1.0-5ab73d26-server.jar file, so, this seems pretty exclusive aand, I kinda don't want to have a virus on my old clanker. Hope somebody resolves where this is in the code and yeah, thanks.

**Comment by JustRed23** [comment]
Seems like a false flag, if you really want to be safe, you can always build from the source. I also noticed that you are using the server jars, no need for those, just download [kettinglauncher](https://github.com/kettingpowered/kettinglauncher) and follow the steps provided in the README

**Comment by JustRed23** [comment]
Ah nvm, read your question wrong

**Comment by JustRed23** [comment]
It does look like a false flag, but if you want to be sure, here is the sample uploaded to virustotal: https://www.virustotal.com/gui/file/3f61cde62e6e95b2efdfbb1985f0fe33363fe65b561508785d87857d3ee89860?nocache=1

**Comment by C0D3-M4513R** [comment]
Well, what antivirus detects this?

**Comment by Epik-B0mber** [comment]
Wow you guys were so fast to respond I was surprised to get up and sit back down to responses.

I just found out about this project about 2 months ago so I was initially using the server .jar until I checked earlier this week and noticed updates were coming through. So I went ahead and got the latest and greatest, and, oddly enough, my computer didn't do jack when I made two test servers with the jar.
One Microsoft update later the Windows Defender (so it's up to date) is now whining about a trojan virus; I've never cared much for what it says before but it's the first time it's straight deleted files off of my computer so I wanted to voice my concern.

Thank you very much for checking!
